Resources/Private/Partials/Cart/ProductForm/ProductList.html

```diff
@@ -3,7 +3,7 @@
       data-namespace-typo3-fluid="true">
 
 <f:section name="withoutVariant">
-    <tr class="{f:if(condition: product.quantityIsInRange, then: '', else: 'danger')}">
+    <tr class="{f:if(condition: '!{product.quantityBelowRange} || !{product.quantityAboveRange}', then: '', else: 'danger')}">
```

Comment:
There is a method `isQuantityInRange`.

```suggestion
    <tr class="{f:if(condition: product.quantityInRange, then: '', else: 'danger')}">
```

should work. So we can use the more readable syntax in condition.